APE18.rst

```diff
@@ -0,0 +1,69 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+After the release of astropy 4.0 APE 10 no longer specifies the supported versions of CPython.
+Since APE 10 was written the wider scientific Python community have written and largely adopted `NEP 29 <https://numpy.org/neps/nep-0029-deprecation_policy.html>`__ as a policy for supported versions of both CPython and Numpy.
+This APE proposes that Astropy follow the projects who have already adopted this policy.
```

Comment:
Adopt this policy following other projects in the scientific python ecosystem or something like that? (That is we aren't pinning our policy to other packages)